pymongo/pool.py

```diff
@@ -752,6 +752,7 @@ def __init__(
         self.active = False
         self.last_timeout = self.opts.socket_timeout
         self.connect_rtt = 0.0
+        self.logical_session_timeout_minutes: int = 0
```

Comment:
Let's default this to None or avoid confusing the 0 and None cases. (It's technically possible to set logical_session_timeout_minutes=0.)